concept_r_python_cheatsheet.Rmd

```diff
@@ -0,0 +1,185 @@
+# draw graphs using python and R
+
+Shuyue Xu
+
+```{r, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+## things that need to be done before fitting models
+##### 1.check unique of the values
```

Comment:
Please fix so hashtag headers are not followed by numbers since the hashtags will create numbered subheadings